codecs/avif/Makefile

```diff
@@ -50,7 +53,8 @@ $(OUT_ENC_JS): $(OUT_ENC_CPP) $(CODEC_DIR)/CMakeLists.txt $(LIBAOM_DIR)/CMakeLis
 		LIBAVIF_FLAGS="-DAVIF_CODEC_AOM_DECODE=0 -DAVIF_LOCAL_LIBSHARPYUV=ON"
 
 # MT-Encoding
-$(OUT_ENC_MT_JS): $(OUT_ENC_CPP) $(CODEC_DIR)/CMakeLists.txt $(LIBAOM_DIR)/CMakeLists.txt $(LIBSHARPYUV)
+$(OUT_ENC_MT_JS): $(OUT_ENC_CPP) $(CODEC_DIR)/CMakeLists.txt $(LIBAOM_DIR)/CMakeLists.txt $(LIBSHARPYUV_MT)
+	mkdir -p $(LIBWEBP_DIR)/build && cp $(LIBSHARPYUV_MT) $(LIBSHARPYUV)
```

Comment:
This looks like it could race with `$(OUT_ENC_JS)` in a multi-job build, unless they're otherwise serialized.

Reply:
Good call. I've added a dependency on `$(OUT_ENC_JS)` to ensure there's no conflicts there.

It's the best we can do for now, unless there's a friendlier way to pass the custom `libsharpyuv` path to the build.

Reply:
> Good call. I've added a dependency on $(OUT_ENC_JS) to ensure there's no conflicts there.

It's a minor detail, but an [order-only dependency](https://www.gnu.org/software/make/manual/make.html#Prerequisite-Types) might be enough here `(: ... | $(OUT_ENC_JS)`.

> It's the best we can do for now, unless there's a friendlier way to pass the custom `libsharpyuv` path to the build.

Not without modifying the [LocalLibsharpyuv.cmake](https://github.com/AOMediaCodec/libavif/blob/98ce107ee731a80894e389b26f972e48e82a259d/cmake/Modules/LocalLibsharpyuv.cmake#L3) file.

Reply:
Thank you! I wasn't aware of that. I believe that's a nicer solution, I'll update.